canonical-web-and-design/maas.io
# Fix contact us

## Done
- Add SECRET_KEY to env
- Add the updated global-nav module
- Remove the underscore from contact-us partial 
- Tidy up some old unused js files

## QA
- Open the site
- Click Contact us in the header
- See the model form opens and once complete submits

## Issue / Card
Fixes https://github.com/canonical-web-and-design/maas.io/issues/476
